style-dictionary/classic/spacing.ts

```diff
@@ -40,6 +40,10 @@ const tokens: StyleDictionary.SpacingDictionary = {
   spaceLayoutContentHorizontal: '{spaceScaled2xXxxl}',
   spaceModalContentBottom: '{spaceScaledL}',
   spaceOptionIconBigTop: '{spaceXxxs}',
+  spacePanelContentBottom: '{spaceScaledXxxl}',
```

Comment:
Classic takes the Visual Refresh values as defaults (this is done with `merge` below), so I think only the tokens that differ from Visual Refresh need to be specified here.

Reply:
While that's true, these might change in the future (I have another PR prepared where this is the case). So I think a more explicit approach is also fine.